base/shared/src/main/scala/scalaz/Prelude.scala

```diff
@@ -119,13 +119,12 @@ trait BaseDataAliases { self: BaseData =>
 }
 
 trait AllFunctions
-    extends data.VoidFunctions
-    with data.DisjunctionFunctions
-    with data.MaybeFunctions
-    with data.KleisliFunctions
-    with ct.InvariantFunctorFunctions
-    with ct.PhantomFunctions
-    with ct.TraversableFunctions
+    extends AnyRef
```

Comment:
What's the benefit of extending `AnyRef` here?

Reply:
If we add a new package that is alphabetically before `algebra`, we don't need to change the line `algebra.AlgebraFunctions` to add it here.